cmd/runtimetest/main.go

```diff
@@ -191,6 +192,81 @@ func validateSysctls(spec *rspec.Spec) error {
 	return nil
 }
 
+func mountMatch(specMount rspec.Mount, sysMount rspec.Mount) error {
+	if specMount.Destination != sysMount.Destination {
+		return fmt.Errorf("mount destination expected: %v, actual: %v", specMount.Destination, sysMount.Destination)
+	}
+
+	if specMount.Type != sysMount.Type {
+		return fmt.Errorf("mount %v type expected: %v, actual: %v", specMount.Destination, specMount.Type, sysMount.Type)
+	}
+
+	if specMount.Source != sysMount.Source {
+		return fmt.Errorf("mount %v source expected: %v, actual: %v", specMount.Destination, specMount.Source, sysMount.Source)
+	}
+
+	optMap := make(map[string]bool)
+	for _, opt := range sysMount.Options {
+		optMap[opt] = true
+	}
+	for _, option := range specMount.Options {
+		if _, ok := optMap[option]; !ok {
+			return fmt.Errorf("mount %v option %v does not exist", specMount.Destination, option)
+		}
+	}
+
+	return nil
+}
+
+func containString(opts []string, str string) bool {
+	for _, opt := range opts {
+		if opt == str {
+			return true
+		}
+	}
+
+	return false
+}
+
+func validateMounts(spec *rspec.Spec) error {
+	fmt.Println("validating mounts")
```

Comment:
We should probably be using a logging framework for these so folks can adjust the verbosity.